Core/Utilities/TimeLeft/TimeLeft.py

```diff
@@ -76,10 +85,23 @@ def getTimeLeft( self, cpuConsumed = 0.0 ):
       return resourceDict
 
     resources = resourceDict['Value']
-    if not resources['CPULimit'] or not resources['WallClockLimit']:
+    self.log.verbose( resources )
+    if not resources['CPULimit'] and not resources['WallClockLimit']:
       # This should never happen
       return S_ERROR( 'No CPU or WallClock limit obtained' )
 
+    # if one of CPULimit or WallClockLimit is missing, compute a reasonable value
+    if not resources['CPULimit']:
+      resources['CPULimit'] = resources['WallClockLimit']
```

Comment:
OK fine... Indeed I had it real in a local branch here, but I will let you PR to be merged first otherwise we shall have conflicts and Andrei will not be happy...
At least the functionality is not changed... 
Review OK for me then!